app/backend/core/imageshelper.py

```diff
@@ -1,8 +1,12 @@
 import base64
+import math
 import os
+import re
+from io import BytesIO
 from typing import Optional
 
 from azure.storage.blob.aio import ContainerClient
+from PIL import Image
```

Comment:
This does add a pillow dependency :(

Reply:
We do already have pillow in scripts, so not a big deal, but I'd feel better about it if it was required for non-vision use cases. I guess we already have precedent for that though, with all our authentication related dependencies.